bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -124,7 +124,7 @@ class App extends Component {
 
     return (
       <Resizable
-        minWidth="10%"
+        minWidth="13%"
```

Comment:
Why don't you add a `defaultSize` like in chat? in the your PR the width resize is limited to grow only.

Dev2a:
![current-dev2a](https://user-images.githubusercontent.com/15806883/40422706-8d6e343a-5e66-11e8-8d4f-1514bbcb88a5.gif)

Your PR: 
![pr](https://user-images.githubusercontent.com/15806883/40422756-aa54a44e-5e66-11e8-8e86-5ae0c0cb8492.gif)

Reply:
Because if I put static default size % inside resizable tag, the bigger the screen size is, the more the user-list has wasteful space as default

Reply:
So I just adjusted the min-width % bigger because when the user-list width is automatically resized according to the screeen size, some screen sizes of some laptops make their “public chat” using ellipsis like “public ch..” as default. And, could you tell me what screen size you used to test it to let me check it?

Reply:
Did you use same screen size for testing? If then, Could you tell me the screen size of your workstation?